### ISSUE TITLE
org-html-export-to-html issues?

### ISSUE THREAD

**Issue by grettke** [alpaker/fill-column-indicator#45]
Hi,

I'm working with Org mode 8.2.5h and during export to html with fci-mode enabled I get the characters &#57344;&#57345;&#57345; included in the export which is not expected.

Totally understood that this is not the org-model list, I'm just looking to talk with other fci users who also use org-mode to find out if you faced this and how you resolved it and also wondering where/how we might contribute to tracking down any unexpected behavior.

**Comment by DesmondHayes** [comment]
I use the following code in my Emacs config to fix this issue for now.
```elisp
(defun fci-mode-override-advice (&rest args))
(advice-add 'org-html-fontify-code :around
            (lambda (fun &rest args)
              (advice-add 'fci-mode :override #'fci-mode-override-advice)
              (let ((result  (apply fun args)))
                (advice-remove 'fci-mode #'fci-mode-override-advice)
                result)))
```

**Comment by alpaker** [comment]
The functionality provided by this package has now been implemented natively in Emacs as `display-fill-column-indicator-mode`, available as of version 27.0.90. Unless you are forced to use an older Emacs, you should use the native implementation instead of `fci-mode`. The former is unaffected by most of the compatibility issues that this add-on has.
